addons/website_sale_mondialrelay/static/src/js/checkout.js

```diff
@@ -88,12 +104,25 @@ websiteSaleCheckoutWidget.include({
                     }, 10000);
                 },
             };
-            this.$modal_mondialrelay.find('#o_zone_widget').MR_ParcelShopPicker(params);
-            this.$modal_mondialrelay.modal('show');
-            this.$modal_mondialrelay.find('#o_zone_widget').trigger("MR_RebindMap");
+
+            scriptJqueryEl.onload = () => {
+                const scriptEl = document.createElement("script");
+                scriptEl.src =
+                    "https://widget.mondialrelay.com/parcelshop-picker/jquery.plugin.mondialrelay.parcelshoppicker.min.js";
+                iframeEL.contentDocument.head.append(scriptEl);
+                scriptEl.onload = () => {
+                    const zoneWidgetEl =
+                        iframeEL.contentDocument.querySelector("#o_iframe_zone_widget");
+                    const $ = iframeEL.contentWindow.$;
+                    $(zoneWidgetEl).MR_ParcelShopPicker(params);
+                    this.modalMondialrelayBS = new Modal(this.modal_mondialrelay);
```

Comment:
the first letter of the word "relay" must be in uppercase in all variable names :wink: 
```suggestion
                    this.modalMondialRelayBS = new Modal(this.modalMondialRelayEl);
```